README.md

```diff
@@ -1,162 +1,138 @@
-# Module for Single Cell Expression Atlas database loading (v0.2.3)
+# Single Cell Expression Atlas database loading module (v1.0.0)
 
-An AtlasProd module for loading scxa-* tables data to postgres >=10. Release v0.2.3 was used for the May 2020 Data release of Singe Cell Expression Atlas.
+A [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc) module for loading experiments to a Postgres 11 
+database. Release v0.4.0 was used for [the October 2022 data release of Singe Cell Expression 
```

Comment:
Since 0.2.3 was used for the May 2020 release, and versioning of this package had been neglected for a while, I thought that the version used in October 2022 would have been something around 0.4.0.